### PR TITLE
Remove Old Post/Preprocessing Folders

### PR DESCRIPTION
**Pull request by davschneller**
Cf. #872 (actually a lot cherry-picked from there), but we don't remove all folders from the preprocessing/postprocessing—only those that are already marked as deprecated in the documentation. Besides, we update the contributing guide a tiny bit.

### CODE REVIEW

File: AUTHORS.txt [hunk 1]
```diff
@@ -18,6 +24,7 @@ The SeisSol group appreciates the work of following contributors and former core
  * Jordan, Kirk - IBM Research
  * Kaeser, Martin - MunichRE
  * Krenz, Lukas - TUM
+ * Kurapati, Vikas - TUM
```
Comment (Thomas-Ulrich) [inline AUTHORS.txt:27]: this list is listing `former core group members`

Comment (vikaskurapati) [reply AUTHORS.txt]: It says `contributors and former core members`, doesn't that mean current contributors and former core members? Or do we mean "former contributors and core members"?

Comment (davschneller) [reply AUTHORS.txt]: Ok; removed myself and Vikas then. :)
For now

File: README.md [hunk 2]
```diff
@@ -9,7 +15,7 @@
 SeisSol is a scientific software for the numerical simulation of seismic wave phenomena and earthquake dynamics. It is based on the discontinuous Galerkin method combined with ADER time discretization. Visit our [official website](http://www.seissol.org/) and also read our [documentation](https://seissol.readthedocs.io) in order to learn more about SeisSol. Our [discussion forum](https://github.com/SeisSol/SeisSol/discussions) can help you to communicate with SeisSol developers or other users, share your ideas, ask questions, etc.
 
 ### Note
-SeisSol is still under heavy development and comes without any guaranteed funcitonality. At the moment we can only provide very limited support for general users. 
+SeisSol is still under heavy development and comes without any guaranteed funcitonality. At the moment we can only provide very limited support for general users.
```
Comment (Thomas-Ulrich) [inline README.md:18]: functionality

**Review by Thomas-Ulrich — APPROVED** [review]
see also comment

File: AUTHORS.txt (hunk 1 above)
Comment (Thomas-Ulrich) [reply AUTHORS.txt]: you are include in `The SeisSol group`

File: README.md (hunk 2 above)
Comment (davschneller) [reply README.md]: Interesting—that typo's been in there for three years now.

**Pull request merged**